bindings_generator/src/main.rs

```diff
@@ -1,30 +1,31 @@
+use std::path::Path;
+
 fn main() {
-    let out_path = std::env::current_dir().unwrap();
-    let header = out_path.join("wrapper.h").to_str().unwrap().to_string();
-    let out_file = out_path.join("../src/bindings.rs");
+    let header = Path::new(env!("CARGO_MANIFEST_DIR"))
+        .join("wrapper.h")
```

Comment:
Just like `-I./vendor/` isn't using `CARGO_MANIFEST_DIR` currently, this also might not need to because the build script always runs with the working directory set to the crate root (the `bindings_generator/` folder).

Reply:
This was causing issues so I added the cargo manifest dir